packages/theme-language-server-common/src/server/startServer.spec.ts

```diff
@@ -58,7 +58,7 @@ describe('Module: server', () => {
       }
     });
 
-    fileTree = { 'snippets/code.liquid': fileContents };
+    fileTree = { '.theme-check.yml': '', 'snippets/code.liquid': fileContents };
```

Comment:
Since `Application Support/Code/User/snippets` exists and since `Application Suport/Code/User/settings.json` is a file you're likely to edit, we were inferring `Application Support/Code/User` to be the root of a theme. That was not very good. So I changed the algo to look for `snippets` _and_ `assets` (as well as looking for `.theme-chekc.yml`).

For the unit tests, it was easiest to simply put a `.theme-check.yml`